types/request.d.ts

```diff
@@ -14,6 +15,11 @@ export interface RequestGenericInterface {
   Headers?: RequestHeadersDefault;
 }
 
+export interface ValidationFunction {
```

Comment:
@Eomm i actually wonder if we should have used to export the ValidationFunction in ajv-compiler

Reply:
nevermind